tests/h/views/groups_test.py

```diff
@@ -55,7 +55,7 @@ def call_on_success(request, form, on_success, on_failure):
 
         controller.post()
 
-        assert group_service.create.call_args_list == [
+        assert group_service.create_private_group.call_args_list == [
```

Comment:
Can you help me understand the difference between this pattern and `assert_called_once_with`?

Reply:
This was preexisting code, but it looks to me as if `assert_called_once_with()` would indeed be a better way to do this.